front/components/items/AppointmentApproval/_style.scss

```diff
@@ -0,0 +1,13 @@
+.appointmentApproval {
```

Comment:
i remember that you were splitting names by - or _ in classes. Why do you use camelCase rn?